src/main/java/uk/gov/companieshouse/acsp/model/dto/AcspDataDto.java

```diff
@@ -1,30 +1,64 @@
-package uk.gov.companieshouse.acsp.model;
+package uk.gov.companieshouse.acsp.model.dto;
 
-import org.springframework.data.annotation.Id;
-import org.springframework.data.mongodb.core.mapping.Document;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.companieshouse.acsp.model.enums.AMLSupervisoryBodies;
+import uk.gov.companieshouse.acsp.model.enums.RoleType;
+import uk.gov.companieshouse.acsp.model.enums.TypeOfBusiness;
 
 import java.util.Date;
 import java.util.Map;
 
-@Document(collection = "applications")
-public class AcspData {
-    @Id
+
+public class AcspDataDto {
+
+    @JsonProperty("id")
     private String id;
+
+    @JsonProperty("first_name")
     private String firstName;
+
+    @JsonProperty("last_name")
     private String lastName;
-    private Address[] addresses; // need to match with web
+
+    @JsonProperty("addresses")
+    private AddressDto[] addresses; // need to match with web
+
+    @JsonProperty("typeOfBusiness")
     private TypeOfBusiness typeOfBusiness;
+
+    @JsonProperty("roleType")
     private RoleType roleType;
+
+    @JsonProperty("dateOfBirth")
     private Date dateOfBirth;
+
+    @JsonProperty("verified")
     private Boolean verified;
-    private Nationality[] nationality; // no need of array
+
+    @JsonProperty("nationality")
+    private NationalityDto[] nationality; // no need of array
+
+    @JsonProperty("countryOfResidence")
     private String countryOfResidence;
+
+    @JsonProperty("businessName")
     private String businessName;
-    private SectorOfWork workSector;
+
+    @JsonProperty("workSector")
+    private SectorOfWorkDto workSector;
+
+    @JsonProperty("amlSupervisoryBodies")
     private Map<AMLSupervisoryBodies, Integer> amlSupervisoryBodies;
```

Comment:
Will the AML body number always be an integer?

Reply:
Actually not... it has to be a String... but it will be covered in the refactoring ticket.